### PR TITLE
Windows/Linux build

### PR DESCRIPTION
**Pull request by ianomad**
This at least doesn't stop development on Windows locally. Fixed after @lvxv

### CODE REVIEW

**Comment by victorNewRelic** [comment]
Hi @ianomad,
I commented on the other [Windows PR](https://github.com/newrelic/go-agent/pull/6#issuecomment-243872967) that you referenced saying that we're going to address support for additional operating systems, including Windows, once we finish other critical features that are important for many customers.

Until then, we'll label this PR to reflect that it's under consideration.

Thanks for taking the time to provide this PR.

Best regards,
Victor Soares

**Comment by nathany** [comment]
@ianomad @lvxv Thanks for the patch.

I'm trying it out on the most recent go-agent codebase.

From $GOPATH/src/github.com/newrelic/go-agent>
```
git checkout -b windows
hub am -3 https://github.com/newrelic/go-agent/pull/10
GOOS=windows go build
```
👍 

I'm going to vendor this branch until Windows support gets official. #8

File: internal/utilities_linux.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package internal
```
Comment (nathany) [inline internal/utilities_linux.go]: To continue using this code on darwin (macOS) and BSD, this file should not be named `_linux.go`. Instead use a [build constraint](https://golang.org/pkg/go/build/) comment:

```
// +build !windows
```

**Comment by willnewrelic** [comment]
Hi

Version 1.5.0 was just released with Windows support!  Thanks for your interest and help @ianomad

**Comment by ianomad** [comment]
@willnewrelic awesome!

**Pull request closed without merging**